lib/index.js

```diff
@@ -17,25 +21,25 @@ for(const module of contextModules) {
   const {contexts, constants: contextConstants} = require(module);
   for(const c in contextConstants) {
     const contextUrl = contextConstants[c];
-    bedrockConstants[c] = contextUrl;
-    bedrockConstants.CONTEXTS[contextUrl] = contexts.get(contextUrl);
+    dlc.urls[c] = contextUrl;
+    dlc.documents.set(contextUrl, contexts.get(contextUrl));
```

Comment:
If we use a Map here then we can't convert configs to/from JSON. We used to use this for reasons that I can't recall... and this might be the first case of us deviating so I'd rather us not and just use a regular `object`.

Reply:
I like the maps, but hopefully we can sidestep and unknowns by not using the config at all as was suggested below.

Reply:
This is no longer a consideration.